libraries/exoplayer/src/main/java/androidx/media3/exoplayer/video/VideoRendererEventListener.java

```diff
@@ -205,6 +218,16 @@ public void droppedFrames(int droppedFrameCount, long elapsedMs) {
       }
     }
 
+    /** Invokes {@link VideoRendererEventListener#onConsecutiveDroppedFrames(int, long)}. */
+    public void consecutiveDroppedFrames(int consecutiveDroppedFrameCount, long elapsedMs) {
+      if (handler != null) {
+        handler.post(
+            () ->
+                castNonNull(listener)
+                    .onConsecutiveDroppedFrames(consecutiveDroppedFrameCount, elapsedMs));
```

Comment:
Can you add a test to `MediaCodecVideoRendererTest` to make sure that provided a very late buffer that is dropped that this event is invoked? You can look at `MediaCodecVideoRendererTest::render_withLateBuffer_dropsBuffer`. May also be helpful to have unit tests for the event testing the output for if dropped frames occur prior to any successful renders and between successful renders.

Reply:
Added tests covering both the API being invoked and only being invoked when there are enough frame drops.